backend/src/controllers/tour.js

```diff
@@ -106,6 +106,20 @@ const search = (req, res) => {
     //     }));
 };
 
+const getParticipants = (req, res) => {
+    TourModel.findById(req.params.id).populate('participants').exec()
```

Comment:
I am not really sure, how these relationships work in mongoose, but at least you are not returning participants, but tour with participants. So please, return just participants.

Reply:
Changed to return only the tour participants